lib/langchain.rb

```diff
@@ -51,7 +51,7 @@ module Langchain
   autoload :Loader, "langchain/loader"
   autoload :Data, "langchain/data"
   autoload :Conversation, "langchain/conversation"
-  autoload :ConversationMemory, "langchain/conversation_memory"
+  autoload :BaseConversationMemory, "langchain/base_conversation_memory"
```

Comment:
Why do we need to change the class name? I think `MyConversationMemory < Langchain::ConversationMemory` actually makes sense.